packages/salesforcedx-vscode-core/package.nls.ja.json

```diff
@@ -70,5 +71,6 @@
   "stop_apex_debug_logging": "SFDX: Replay Debugger 用の Apex デバッグログを無効化",
   "telemetry_enabled_description": "Salesforce による利用状況データの収集を有効化するかどうかを指定します。この設定で有効化していても、VS Code 全体の設定 `telemetry.enableTelemetry` が優先されます。",
   "visualforce_generate_component_text": "SFDX: Visualforce コンポーネントを作成",
-  "visualforce_generate_page_text": "SFDX: Visualforce ページを作成"
+  "visualforce_generate_page_text": "SFDX: Visualforce ページを作成",
+  "rename_lightning_component_text": "SFDX: Rename Component"
```

Comment:
Interesting that this used to be only present in the English file and not the Japanese one - good catch!